stdlib/public/Concurrency/Executor.swift

```diff
@@ -625,9 +632,8 @@ extension Task where Success == Never, Failure == Never {
   /// executor is a fatal error.
   @available(SwiftStdlib 6.2, *)
   public static var defaultExecutor: any TaskExecutor {
-    if _defaultExecutor == nil {
-      _defaultExecutor = DefaultExecutorFactory.defaultExecutor
-    }
+    // It would be good if there was a Swift way to do this
+    _createDefaultExecutorsOnce()
```

Comment:
You can do a `swift_once` in Swift with a dummy global variable (I think it can be `Void`) with an initializer that does whatever it is you want it to do, then do a dummy access like `_ = doTheSetup` to trigger it. This is fine too of course.

Reply:
:-D I hadn't considered that, though it would still be good to have less round-about way to do it.

Reply:
This was actually official advice given when `dispatch_once` was made unavailable way back in Swift 3: https://www.swift.org/migration-guide-swift3/